docs/root/intro/version_history.rst

```diff
@@ -23,6 +23,8 @@ Version history
   See `#3611 <https://github.com/envoyproxy/envoy/issues/3611>`_ for details.
 * http: augmented the `sendLocalReply` filter API to accept an optional `GrpcStatus`
   value to override the default HTTP to gRPC status mapping.
+* http: no longer close the TCP connection when a HTTP/1 request is retried due
+  to a 5xx/499 response with empty body.
```

Comment:
really small nit:
due  to a 5xx/499 response with empty body. -> due to a response with empty body.
since we cover a few other cases?  Otherwise LGTM!

Reply:
oh, I see, it does! TIL `x-envoy-retriable-status-codes`